### PR TITLE
New login screen UI

### LINKED ISSUE #30: Changing login screen design according to new design model
### I understand and confirm the below statements

- [x] I have read the [Wiki](https://github.com/iiitv/club_calender/wiki) and I understand the concept and structure of the app.
- [x] The functionality has not already been covered in an existing issue.
- [x] Assign this issue to me (I want to work on this).
	- [ ] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within 3 days.

### PR DESCRIPTION
**Pull request by Monik09**
Fixes #30 

flutter_01

### CODE REVIEW

**Comment by thecodepapaya** [comment]
@Chirag-Mathur take a look at the design implementation and let us know if it looks alright.
@aashutoshrathi please let us know what you think 🙌

ps: it's `calendar` and not `calender` 😛
pps: @Chirag-Mathur please correct the designs accordingly and @Monik09 create issues if you see similar mistakes in the source code as well

File: lib/ui/login_screen.dart [hunk 1]
```diff
@@ -1,96 +1,114 @@
...
+                        Flexible(
                           child: Text(
-                            " Sign In with Google",
-                            style: GoogleFonts.roboto(
-                              color: Colors.white,
-                              fontSize: deviceHeight * 0.02711292,
-                            ),
+                            " Log in with Google",
+                            style: GoogleFonts.montserrat(
+                                fontSize: aspectRatio * 45,
```
Comment (thecodepapaya) [inline lib/ui/login_screen.dart:109]: Keep the font sizes fixed irrespective of the screen size. It gets messy when we test applications on devices having different aspect ratios. However, it's okay to use this method for other visual components which need to be dynamic.

Define different TextStyles in `styles.dart` with all the necessary parameters like font weight, font size, font colour etc.
You should have a fixed set of consistent font styles defined in the file which could be used directly in the UI components.

File: lib/ui/login_screen.dart [hunk 2]
```diff
@@ -1,96 +1,114 @@
...
+                        child: Text(
+                          "CLUB CALENDER",
+                          // softWrap: true,
+                          textAlign: TextAlign.center,
+                          style: GoogleFonts.montserrat(
+                              color: Styles.buttonColor,
+                              fontSize:
+                                   aspectRatio*
+                                     75,
+                              fontWeight: FontWeight.w400),
```
Comment (thecodepapaya) [inline lib/ui/login_screen.dart:66]: Same for this

**Review by thecodepapaya — CHANGES_REQUESTED** [review]

File: lib/ui/login_screen.dart (hunk 1 above)
Comment (Monik09) [reply lib/ui/login_screen.dart:109]: I create an issue for extra info of text styles in styles.dart file

**Comment by aashutoshrathi** [comment]
> @Chirag-Mathur take a look at the design implementation and let us know if it looks alright.
> @aashutoshrathi please let us know what you think 🙌
> 
> ps: it's `calendar` and not `calender` 😛
> pps: @Chirag-Mathur please correct the designs accordingly and @Monik09 create issues if you see similar mistakes in the source code as well

The screenshot looks pretty neat as per designs, Do we have an temp APK here to try out?

**Comment by Chirag-Mathur** [comment]
> @Chirag-Mathur take a look at the design implementation and let us know if it looks alright.

The screenshot looks good to me.  👍  Make sure to look for the design after pressing the login button.

**Comment by thecodepapaya** [comment]
#### The button design is not matching with the mock designs. There is an overflow and the button content is flowing to the next line.
#### Also, the shadows in actual implementation look slightly darker.

|Design|Actual|
|---|---|
|||

#### The pressed button design does not match with the one shown in the designs. Please fix this.

|Design|Actual|
|---|---|
|||

**Review by thecodepapaya — CHANGES_REQUESTED** [review]
Take a look at the difference in the designs and fix the spelling errors.

**Comment by Monik09** [comment]
> #### The button design is not matching with the mock designs. There is an overflow and the button content is flowing to the next line.
> #### Also, the shadows in actual implementation look slightly darker.
> Design	Actual
> 	
> #### The pressed button design does not match with the one shown in the designs. Please fix this.
> Design	Actual
> 	

Yes @thecodepapaya sir, i am currently working on the button and also have changed the spellings.

**Comment by Monik09** [comment]
Without pressing the button<a href="Unpressed LoginButton">
</a>
<br>
After pressing the button
<a href="Pressed LoginButton"></a>

**Review by thecodepapaya — APPROVED** [review]
LGTM

**Pull request merged**